src/main/java/team/asd/dao/ImplPaymentTransactionDao.java

```diff
@@ -1,27 +1,32 @@
 package team.asd.dao;
 
+import lombok.AllArgsConstructor;
 import org.springframework.stereotype.Repository;
 import team.asd.entity.PaymentTransaction;
+import team.asd.mapper.PaymentTransactionMapper;
 
 @Repository
-public class ImplPaymentTransactionDao implements PaymentTransactionDao{
+@AllArgsConstructor
+public class ImplPaymentTransactionDao implements PaymentTransactionDao {
```

Comment:
@Sem4enko wrong class naming
Please read this: https://thomaspoignant.medium.com/java-how-to-name-interface-and-implementor-94c0fa564b87